test/junit/scala/collection/FactoriesTest.scala

```diff
@@ -16,15 +15,15 @@ class FactoriesTest {
     def cloneCollection[A, C](xs: Iterable[A])(implicit bf: BuildFrom[xs.type, A, C]): C =
       bf.fromSpecific(xs)(xs)
 
-    Assert.assertEquals("ArrayBuffer", cloneCollection(seq).collectionClassName)
+    Assertions.assertEquals("ArrayBuffer", cloneCollection(seq).collectionClassName)
   }
 
   @Test def factoryIgnoresSourceCollectionFactory(): Unit = {
 
     def cloneElements[A, C](xs: Iterable[A])(cb: Factory[A, C]): C =
       cb.fromSpecific(xs)
 
-    Assert.assertEquals("List", cloneElements(seq)(Seq).collectionClassName)
+    Assertions.assertEquals("List", cloneElements(seq)(Seq).collectionClassName)
```

Comment:
future cleanup for someone: we're already importing `assertEquals`, so no reason to call it as `Assertions.assertEquals`